qemu/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM amd64/alpine:3.13
 
 ENV LIBISCSI_VERSION 1.19.0
-ENV QEMU_VERSION 4.1.0
+ENV QEMU_VERSION 6.0.0-rc1
```

Comment:
Don't forget to reflect the change in README:
* reference to https://wiki.qemu.org/ChangeLog/6.0
* specify `30 Mar 2021` as `Release date`